spec/unit/commands/plans_command/import/validation_plan_step_spec.rb

```diff
@@ -0,0 +1,355 @@
+RSpec.describe ThreeScaleToolbox::Commands::PlansCommand::Import::ValidatePlanStep do
+  let(:artifacts_resource) { { } }
+  let(:service) { instance_double(ThreeScaleToolbox::Entities::Service) }
+  let(:backend_01) { instance_double(ThreeScaleToolbox::Entities::Backend, 'backend_01') }
+  let(:backend_02) { instance_double(ThreeScaleToolbox::Entities::Backend, 'backend_02') }
+  let(:backend_usage_01) { instance_double(ThreeScaleToolbox::Entities::BackendUsage, 'backend_usage_01') }
+  let(:backend_usage_02) { instance_double(ThreeScaleToolbox::Entities::BackendUsage, 'backend_usage_02') }
+  let(:backend_usage_list) { [backend_usage_01, backend_usage_02] }
+  let(:context) do
+    {
+      service: service,
+      artifacts_resource: artifacts_resource
+    }
+  end
+  subject { described_class.new(context).call }
+
```

Comment:
Maybe a test case with a successful validation would be good

Reply:
done

Reply:
:+1: